Energy Saver/Model/Serialization.cs

```diff
@@ -7,10 +7,14 @@ public static class Serialization
 {
     private static string path = "Resources/Taxes.json";
 
-    private static JsonSerializerSettings serializerSettings = new JsonSerializerSettings
+    private static JsonSerializerSettings GetSerializerSettings<T>(T contractResolver) where T : IContractResolver
```

Comment:
Is this more useful than just creating a JsonSerializerSettings variable?

Reply:
We get to use generic types in this case.